build error

using git rev 5415eaa22baccd8316730ea1889c25fb901ce187 on debian jessie

```
[ 34%] Building CXX object CMakeFiles/flare.dir/src/ItemManager.cpp.o
/home/sudoman/src/3rd-party/games/flare/flare-engine/src/ItemManager.cpp: In member function ‘void ItemManager::loadItems()’:
/home/sudoman/src/3rd-party/games/flare/flare-engine/src/ItemManager.cpp:226:107: error: cannot pass objects of non-trivially-copyable type ‘std::string {aka class std::basic_string<char>}’ through ‘...’
     infile.error("%s unrecognized at; requires_stat must be one of [physical:mental:offense:defense]\n", s);
                                                                                                           ^
CMakeFiles/flare.dir/build.make:813: recipe for target 'CMakeFiles/flare.dir/src/ItemManager.cpp.o' failed
make[2]: *** [CMakeFiles/flare.dir/src/ItemManager.cpp.o] Error 1
CMakeFiles/Makefile2:60: recipe for target 'CMakeFiles/flare.dir/all' failed
make[1]: *** [CMakeFiles/flare.dir/all] Error 2
Makefile:117: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/src/ItemManager.cpp
+++ b/src/ItemManager.cpp
@@ -223,7 +223,7 @@ void ItemManager::loadItems() {
 			else if (s == "defense")
 				items[id].req_stat.push_back(REQUIRES_DEF);
 			else
-				infile.error("%s unrecognized at; requires_stat must be one of [physical:mental:offense:defense]\n", s);
+				infile.error("%s unrecognized at; requires_stat must be one of [physical:mental:offense:defense]\n", s.c_str());
 			items[id].req_val.push_back(toInt(infile.nextValue()));
 		}
 		else if (infile.key == "bonus") {

